linkerd/app/inbound/src/policy/authorize/tcp.rs

```diff
@@ -122,42 +122,39 @@ where
     }
 
     fn call(&mut self, io: I) -> Self::Future {
-        match self {
-            // If the connection is authorized, pass it to the inner service and stop processing the
-            // connection if the authorization's state changes to no longer permit the request.
-            Self::Authorized(Authorized {
-                inner,
-                client,
-                tls,
-                policy,
-                metrics,
-            }) => {
-                let client = *client;
-                let tls = tls.clone();
-                let mut policy = policy.clone();
-                let metrics = metrics.clone();
-
-                // FIXME increment counter.
```

Comment:
we still need to increment the counter, right?

Reply:
no, it's incremented above.

Reply:
ah, i can't read! nvm.